Generator/Sources/Generator.swift

```diff
@@ -286,6 +297,8 @@ let nameGenerator = IdiomaticSafeNameGenerator(defensive: .init())
 enum GeneratorError: Error {
     case attributeNameNotFound(String)
     case namespaceNameNotFound(String)
+    case invalidAttributeId(String)
```

Comment:
```suggestion
    case invalidAttributeID(String)
```

Reply:
Fixed in: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/19/commits/4807300ea3b3a63190a981ad5dc45963d5c33b3e